src/molecules/Select/index.jsx

```diff
@@ -19,22 +22,22 @@ export const Select = (props) => {
   const handleChange = (selectedOption) => {
     state.selectedOption = selectedOption;
     setSelected(state.selectedOption);
+    props.onSelect(selectedOption.value, props.propName)
```

Comment:
I have questions on the 'generic-ness' of this method call right here. I think this seems specific to Selects used in the context of the PropsTable. If that's true, we can code this a bit more gracefully and lift this out